geppetto.js/geppetto-ui/src/3d-canvas/Canvas.js

```diff
@@ -294,7 +306,42 @@ Canvas.propTypes = {
   /**
    * Options to customize camera
    */
-  cameraOptions: PropTypes.object,
+  cameraOptions: PropTypes.shape({
```

Comment:
well done @paularah , this will save time to all to see what the props are! 👍🏽

Reply:
one small note, the propTypes are currently missing zoomSpeed and panSpeed, rotateSpeed instead is there. Thx